controllers/contact-controller.js

```diff
@@ -201,6 +201,7 @@ function convertContactsToSearchResults(contacts) {
       const { callType, caseInformation } = contact.rawJson;
       const categories = retrieveCategories(caseInformation.categories);
       const counselor = contact.twilioWorkerId;
+      const addedBy = contact.createdBy;
```

Comment:
The only thought I have is that we should send the raw data from DB, and then we can rename as we want in the fronted, so I prefer this to be called `createdBy`. Up to you and the team tho, just sharing a thought.

Reply:
I agree with gian about the naming.